pkg/agent/openflow/client.go

```diff
@@ -536,10 +539,20 @@ func (c *client) InstallPodFlows(interfaceName string, podInterfaceIPs []net.IP,
 	isAntreaFlexibleIPAM := c.connectUplinkToBridge && c.nodeConfig.PodIPv4CIDR != nil && !c.nodeConfig.PodIPv4CIDR.Contains(podInterfaceIPv4)
 
 	localGatewayMAC := c.nodeConfig.GatewayConfig.MAC
-	flows := []binding.Flow{
-		c.featurePodConnectivity.podClassifierFlow(ofPort, isAntreaFlexibleIPAM),
-		c.featurePodConnectivity.l2ForwardCalcFlow(podInterfaceMAC, ofPort),
+
+	var flows []binding.Flow
+	if c.enableMulticluster {
+		// If multi-cluster is enabled, we store podClassifierFlow with a different key in
+		// cache, in order to reconcile this flow easier when the Pod update its labels.
+		podClassifierFlow := c.featurePodConnectivity.podClassifierFlow(ofPort, isAntreaFlexibleIPAM, UnknownLabelIdentity)
+		err := c.addFlows(c.featurePodConnectivity.podCachedFlows, fmt.Sprintf("%s/classifier", interfaceName), []binding.Flow{podClassifierFlow})
```

Comment:
Seems this won't be deleted if the multi-cluster feature is disabled from enabled? will there be any side effects?

Reply:
If feature gates changed, the agent needs to be rebooted to take effect. So everything will be initiated again.